SG14/inplace_function.h

```diff
@@ -190,15 +223,13 @@ class inplace_function<R(Args...), Capacity, Alignment>
     template<
         typename T,
         typename C = std::decay_t<T>,
-        typename = std::enable_if_t<!inplace_function_detail::is_inplace_function<C>::value>
+        typename = std::enable_if_t<
+            !inplace_function_detail::is_inplace_function<C>::value
+            and inplace_function_detail::is_invocable_r<R, C, Args...>::value
```

Comment:
For MSVC compatibility, this `and` needs to be `&&`. (MSVC doesn't support the English spellings of operator tokens, for some reason unknown to me.)

Reply:
Ahh, right forgot that, they do support it but only if you include some extra header, iirc they claim it's for backward compatibility reasons.